### ISSUE TITLE
Preview permanently blanks if you stop looking at the tab

### ISSUE THREAD

**Issue by joelatschool** [Enter-tainer/typst-preview#81]
Using Ubuntu 23.04, deb-packaged vscode. If you switch away from the tab containing the preview such that it is no longer visible, the tab becomes blank when you switch back until you restart the preview. Checking the output, I get
```
[2023-08-11T22:57:27Z INFO  typst_preview] Peer closed WebSocket connection.
[2023-08-11T22:57:31Z INFO  typst_preview] No active clients. Shutting down.
```

**Comment by Enter-tainer** [comment]
when a webview tab become invisible, vscode disconnect its websocket connection. The server has some self destruction strategy. When there is no client for 10s, it shutdown itself.

For this problem there is no good solution if we keep current behavior. We can 

1. increase the self shutdown timeout
2. give a clear warn in webview to tell user to restart preview

**Comment by Enter-tainer** [comment]
i will try change this behavior to avoid early self destruction

**Comment by Enter-tainer** [comment]
fixed by #88, will appear in next release
